frontend/src/metabase/lib/cron.ts

```diff
@@ -29,16 +29,12 @@ export function validateCronExpression(
 ): string | undefined {
   const result = isValidCronExpression(cronExpression, { error: true });
 
-  // cron-expression-validator's typing is not exactly correct
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
+  // @ts-expect-error cron-expression-validator's typing is not exactly correct
   if (result === true) {
     return;
   }
 
-  // cron-expression-validator's typing is not exactly correct
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
+  // @ts-expect-error cron-expression-validator's typing is not exactly correct
```

Comment:
These 2 errors can be fixed by passing generic type to `isValidCronExpression` call, like this: `isValidCronExpression<boolean>`

Reply:
fixed